src/content/en/basic-workflows/z-image.md

```
@@ -78,6 +78,35 @@ Here we split it into the first 50% and the last 50%.
+
+```text
+📂ComfyUI/
+└── 📂models/
+    └── 📂model_patches/
+        └── Z-Image-Fun-Controlnet-Union-2.1.safetensors
+
+```
+
+### workflow
+
+![](https://gyazo.com/1eb558462ba943c91305960b112c6a63){gyazo=image}
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)